app/services/weet.js

```diff
@@ -0,0 +1,14 @@
+const axios = require('axios');
+
+const API_GEEKS = 'https://geek-jokes.sameerkumar.website/api';
+
+const getGeekPhrase = async () => {
+  const config = {
+    method: 'get',
+    url: `${API_GEEKS}?format=json`
+  };
+  const { data: response } = await axios(config);
+  return response;
+};
+
+exports.getGeekPhrase = getGeekPhrase
```

Comment:
Remember error handling and the use of custom errors, this should be inside a try-catch.